bots/reads.ts

```diff
@@ -265,9 +291,9 @@ const handleNew = async (ctx: ReadsContext) => {
   await ctx.reply('what url do you want post?');
 };
 
-const handlePost = async (ctx: ReadsContext) => {
+const handlePost = async (ctx: ReadsContext, config: ReadsConfig) => {
```

Comment:
I'd suggest having the `publish` function throw on error and generate the reply message here instead. this is the "controller" function whereas the `publish` function is really just a client function for sending the content to the endpoint.